new-lamassu-admin/src/pages/Blacklist/BlacklistModal.js

```diff
@@ -74,9 +75,11 @@ const BlackListModal = ({
         </Form>
       </Formik>
       <div className={classes.footer}>
-        <Link type="submit" form="address-form">
-          Blacklist address
-        </Link>
+        <Box display="flex" justifyContent="flex-end">
+          <Link type="submit" form="address-form">
```

Comment:
I'm not really a fan of inlining styles as props. This is an inconsistency we have thoughout some components in our code base. IMO, we should always keep styles contained in styling files

Reply:
Ohh, this box thing is made to simplify flexbox behavior. This is not really inline styling, just props being passed to the component.